backend/src/judge/middleware/middleware.ts

```diff
@@ -0,0 +1,24 @@
+export interface CodeDetail {
```

Comment:
Let's call these information `prototype` instead since its more similar to the details captured in a function prototype.

```suggestion
export interface CodePrototype {
```